consensus/bor/heimdall/client_test.go

```diff
@@ -150,6 +211,26 @@ func TestFetchShutdown(t *testing.T) {
 		}
 	}
 
+	handler.handleFetchMilestone = func(w http.ResponseWriter, _ *http.Request) {
+		time.Sleep(6 * time.Second)
```

Comment:
Can we get rid of sleep completely or at least make it as little as possible?

Reply:
Okay! I will reduce the time.